libraries/botbuilder-dialogs/src/waterfallDialog.ts

```diff
@@ -157,6 +170,16 @@ export class WaterfallDialog<O extends object = {}> extends Dialog<O> {
         return await this.runStep(dc, 0, DialogReason.beginCalled);
     }
 
+    /**
+     * Called when the waterfall dialog is _continued_, where it is the active dialog and the
+     * user replies with a new activity.
+     * @param dc The <see cref="DialogContext"/> for the current turn of conversation.
```

Comment:
I think this works in .NET but not here.

Reply:
You mean  `<see cref="DialogContext"/>`? If that's the case, it's solved.